src/utils/themeDocUtils.js

```diff
@@ -1,149 +1,252 @@
-      type: 'number',
-      defaultValue: 0.3,
-    },
-  },
-  edgeStyle: description => ({
-    'global.edgeSize': {
-      description,
-      type: 'object',
-      defaultValue: `{
-      edgeSize: {
-        none: '0px',
-        hair: '1px',
-        xxsmall: '3px',
-        xsmall: '6px',
-        small: '12px',
-        medium: '24px',
-        large: '48px',
-        xlarge: '96px',
-        responsiveBreakpoint: 'small',
-      },
-    }`,
-    },
-  }),
-  focusStyle: {
-    'global.focus.border.color': {
-      description: 'The border color of the component when in focus.',
-      type: 'string | { dark: string, light: string }',
-      defaultValue: 'focus',
-    },
-    'global.focus.outline.color': {
-      description: 'The outline color around the component when in focus.',
-      type: 'string | { dark: string, light: string }',
-    },
-    'global.focus.outline.size': {
-      description:
-        'The size of the outline around the component when in focus.',
-      type: 'string',
-    },
-    'global.focus.shadow.color': {
-      description: 'The shadow color around the component when in focus.',
-      type: 'string | { dark: string, light: string }',
-      defaultValue: 'focus',
-    },
-    'global.focus.shadow.size': {
-      description: 'The size of the shadow around the component when in focus.',
-      type: 'string',
-      defaultValue: '2px',
-    },
-  },
-  iconColor: {
-    'global.colors.icon': {
-      description: 'The color of a given icon.',
-      type: 'string | { dark: string, light: string }',
-      defaultValue: '{ dark: #f8f8f8, light: #666666 }',
-    },
-  },
+      "medium": {"value": 1536},
+      "large": {}
+    }
+        `}
+        </Markdown>
+      </Example>
+    </PropertyValue>
+  </Property>
+);
+
+BreakpointStyle.propTypes = {
+  description: PropTypes.node.isRequired,
+};
+
+export const DisabledStyle = () => (
+  <Property name="global.control.disabled.opacity">
+    <PropertyValue>
+      <Description>The opacity when a component is disabled.</Description>
+      <Example defaultValue>0.3</Example>
+    </PropertyValue>
+  </Property>
+);
+
+export const EdgeStyle = ({ description }) => (
+  <Property name="global.edgeSize">
+    <PropertyValue>
+      <Description>{description}</Description>
+      <Example>
+        <Markdown>
+          {`
+    {
+      none: '0px',
+      hair: '1px',
+      xxsmall: '3px',
+      xsmall: '6px',
+      small: '12px',
+      medium: '24px',
+      large: '48px',
+      xlarge: '96px',
+      responsiveBreakpoint: 'small'
+    }
+        `}
+        </Markdown>
+      </Example>
+    </PropertyValue>
+  </Property>
+);
+
+EdgeStyle.propTypes = {
+  description: PropTypes.node.isRequired,
+};
+
+export const FocusStyle = () => (
+  <>
+    <Property name="global.focus.border.color">
+      <Description>
+        The border color of the component when in focus.
+      </Description>
+      <PropertyValue>
```

Comment:
Placing the comment here, but it applies to all the examples -- is there any way we can give more of a visual differentiation between the "description" portion of the docs and the "types". For me, it wasn't immediately clear that the second bit of things was telling me what types were accepted. Maybe it would be helpful to have a small heading of "accepted types" or more spacing after the description, etc.

Reply:
In latest push I added additional spacing under the general description